lab-cameron/README.md

```diff
@@ -0,0 +1,26 @@
+![cf](https://i.imgur.com/7v5ASc8.png) lab 14 double resource express/mongo api (Pokémon themed!)
+======
+
```

Comment:
**TODO FOR LOST POINTS**

Update your README to account for your second resource.